src/components/layout/Shell.tsx

```diff
@@ -42,39 +45,38 @@ export function Shell(props: Props) {
           >
             <Image src="/metabase-logo-with-wordmark.svg" />
 
-            <Box className="md:hidden">
+            <Flex display={{ base: "flex", sm: "none" }}>
               <Burger
                 opened={isMobileNavOpen}
                 onClick={toggleMobileNav}
                 aria-label="Toggle navigation"
                 color="#eee"
               />
-            </Box>
+            </Flex>
 
-            <Box className="hidden md:block">
+            <Flex display={{ base: "none", sm: "flex" }}>
```

Comment:
What's the point of using `Flex` instead of `Box` if you're only specifying `display` and controlling the `flex` value manually? Is there anything not-obvious I missed here.

Reply:
@WiNloSt We could've also used `Box` here, this is just to use `Flex` everywhere in the Shell component. I could revert this back to Box.

Reply:
Fixed in https://github.com/metabase/shoppy/pull/68/commits/04e252be32882e22d79e2f33e7fc861a8819b80d